src/api/routes/animals.ts

```diff
@@ -0,0 +1,20 @@
+import express, { Application, NextFunction, Request, Response } from "express";
+const data = require("../../../data/animaldata.json");
+
+const router = express.Router();
+
+//GET request /animals route
+router.get("/", (req: Request, res: Response, next: NextFunction) => {
+    try {
+        console.log(data);
+        res.status(200).json(data);
+    }
+    catch (err) {
+        console.log(err);
+        res.status(500).json({
+            error: err,
+        });
+    }
+});
```

Comment:
Can we please add a route to get a specific animal?

Reply:
Okay, will do that.